docs/architecture/solana/migration/exchange.mdx

```diff
@@ -28,7 +28,7 @@ If an exchange already supports SPL tokens, follow the steps below.
 :::tip SPL token Support
 
 Please refer to the Solana Documentation on how to
-[Add Solana to Your Exchange](https://docs.solana.com/integrations/exchange) for details.
+[Add Solana to Your Exchange](https://solana.com/docs/more/exchange) for details.
```

Comment:
This change is already handled on main. Let's rebase this branch to pull in those changes.